### PR TITLE
new docs

### PR DESCRIPTION
**Pull request by korry8911**

### CODE REVIEW

**Comment by korry8911** [comment]
@srgg

File: docs/using-connector.md [hunk 1]
```diff
@@ -77,6 +78,17 @@ conf.set("spark.riak.connections.max", "50")
 sc = pyspark.SparkContext("spark://127.0.0.1:7077", "test", conf)
 ```
 
+
+## Failover Handling
+ 
+When reading data from Riak, the Spark-Riak connector will run a full bucket query. This query first obtains metadata, called a coverage plan, from Riak about
```
Comment (srgg) [inline docs/using-connector.md]: IMO better to mention what exact Riak is used for Full Bucket Read (Riak KV):

When reading data from Riak **KV**

File: docs/using-connector.md [hunk 2]
```diff
@@ -77,6 +78,17 @@ conf.set("spark.riak.connections.max", "50")
 sc = pyspark.SparkContext("spark://127.0.0.1:7077", "test", conf)
 ```
 
+
+## Failover Handling
+ 
+When reading data from Riak, the Spark-Riak connector will run a full bucket query. This query first obtains metadata, called a coverage plan, from Riak about 
+where the data is located in the cluster. This coverage plan will have a list of nodes were the data lives, but the Spark-Riak connector will use the first entry
```
Comment (aleksey-suprun) [inline docs/using-connector.md]: Actually connector will use all entries for reading all data. The text related to "first entry" should be removed

File: docs/using-connector.md (hunk 1 above)
Comment (mdigan) [inline docs/using-connector.md]: Instead of referring to Riak KV (the product), let's say "When reading data from a Riak KV bucket..."

**Comment by korry8911** [comment]
updated with the suggested changes

**Comment by korry8911** [comment]
@srgg @mdigan @aleksey-suprun

**Comment by aleksey-suprun** [comment]
Looks fine

**Comment by korry8911** [comment]
Should this be merged? @srgg

**Comment by srgg** [comment]
@korry8911 I will merge it tomorrow, when we will complete all the hacks with 1.5.1 release

**Pull request merged**